QGCSetup.pri

```diff
@@ -87,7 +87,7 @@ WindowsBuild {
     }
 
     DEPLOY_TARGET = $$shell_quote($$shell_path($$DESTDIR_WIN\\$${TARGET}.exe))
-    QMAKE_POST_LINK += $$escape_expand(\\n) $$QT_BIN_DIR\\windeployqt --no-compiler-runtime --qmldir=$${BASEDIR_WIN}\\src $${DEPLOY_TARGET}
```

Comment:
What does this change do?

Reply:
Something to do with what to copy and not to copy (system runtime libraries) @MatejFranceskin Do you know why this was necessary?

Reply:
If it has to to with the C Runtime. Was it verified that new versions of installed QGC on a virgin windows machine still work?